metricflow/sql_clients/big_query.py

```diff
@@ -90,10 +110,15 @@ def _create_bq_engine(
            - Google's decision tree (https://google.aip.dev/auth/4110)
 
         Args:
-            query_string: Additional query params to pass to engine construction.
+            query_field_values: The field/value pairs to use in the query string.
             project_id: Project ID listed on the GCP platform.
             password: String representation of keyfile.json.
         """
+        if query_field_values is None:
+            query_field_values = {}
+        query_items = tuple(f"{field}:{value}" for field, value in query_field_values.items())
```

Comment:
Again, why `:` instead of `=` between param key and value?

Reply:
Same as previous comment.